fastapi/fastapi_dispatcher.py

```diff
@@ -115,5 +115,6 @@ def _manage_odoo_env(self, uid=None):
         token = odoo_env_ctx.set(env)
         try:
             yield
+            env.flush_all()
```

Comment:
@grindtildeath Can you add a comment above the new line with the motivation to do it. The aim is to quickly inform people reading the code why the call to `flush_all`  is being made, and to prevent this line from being inadvertently removed during a migration.

Reply:
@lmignon I amended the commit with the code comment to recap why it's done.

Feel free to suggest a rephrase if it doesn't seem clear enough, and thank you for being so reactive :slightly_smiling_face: